.github/workflows/test.yml

```diff
@@ -8,40 +8,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.3, 7.4, 8.0, 8.1]
-        illuminate_version: [6.*, 8.*]
```

Comment:
It would be great if we could support all versions in the same major release (1.x) -- Could you help me understand what might be a breaking change not allowing us to support these still? (I want to continue supporting laravel 6 til EOL, which is in september).

Reply:
> I don't think it' possible to add Laravel 9 support without dropping 7.3 and 7.4 support.

Laravel 6 won't work well on php 8. It's certainly not supported. That's why I believe laravel 6 support was dropped. 

Is there a major issue with just issuing a 2.0 release for this PR? I don't understand the hesitation at doing a split release as has been already mentioned in this PR previously

Reply:
Larastan (Laravel plugin for PHPStan) also released a new major version for Laravel 9.

> Main reason we have a major version increase is we dropped support for Laravel versions older than 9. Because now all the Collection stubs Larastan had are moved to the Laravel core!

Reply:
My hesitation is ust more maintenance overhead -- I would *think* we could solve that by adjusting the test matrix to not run laravel 6 on php 8 (like we are already doing with other tests) .

It's not a blocker, would just like to understand what exactly is causing the need for multiple versions